pkg/config/config.go

```diff
@@ -58,7 +58,31 @@ type HTTPConfig struct {
 
 type Converter func(s *Serialized, cfg *Controller) (*Controller, error)
 
-func ToController(s *Serialized, cfg *Controller) (*Controller, error) {
+// Returns the important fields of the config in a string form
+func (c *Controller) ToString() string {
+	return fmt.Sprintf("enabled=%t "+
+		"endpoint=%s "+
+		"interval=%s "+
+		"username=%t "+
+		"token=%t "+
```

Comment:
I know we had it before, but should we log this? :)

Reply:
nope, we probably shouldn't :D

Reply:
but we only log if we have a token or not. (`len(c.Token) > 0`)
so its fine